README.md

```diff
@@ -149,23 +150,60 @@ the creation of a new management pack project.
 7. `Enter a path to the Management Pack icon file, or leave blank for no icon`
 
    The icon is used in the VMware Aria Operations UI if present. If it is not present, a default icon will be used. The icon file must be
-   png format and 256x256 px. An icon file can be added later by copying the icon to the root project directory and
+   PNG format and 256x256 pixels. An icon file can be added later by copying the icon to the root project directory and
    setting the value of the `"pak_icon"` key to the icon's file name in the `manifest.txt` file.
 
-8. `Select a language for the adapter. Supported languages are [...]`
-
-   Supported languages are listed. Once selected, the project will be generated, including a template adapter in the
-   selected language. The template adapter collects several objects and metrics from the container that the adapter 
-   is running in, and can be used as a starting point for creating a new adapter.
-
-For complete documentation of the `mp-init` tool see the [MP Initialization Tool Documentation](doc/mp-init.md).
-
+[//]: # (TODO: Add this section back when we support it)
+[//]: # (8. `Select a language for the adapter`)
+[//]: # (  selected language. The template adapter collects several objects and metrics from the container that the adapter)
+[//]: # (  Once selected, the project will be generated, including a template adapter in the)
+[//]: # (  is running in, and can be used as a starting point for creating a new adapter.)
+
+For complete documentation of the `mp-init` tool including an overview of its output, see the [MP Initialization Tool Documentation](doc/mp-init.md).
+
+### Template Adapter
+The template adapter has comments throughout the code to help new users understand the process of creating their own adapter using the existing code. For additional guidance creating adapters see our [Walkthroughs](../README.md#walkthroughs) section.
+The code inside `adapter.py` executed by an HTTP sever that runs inside container every time a test collection, or a collection is made[^1]. The HTTP server sends the information returned by the executable back to VMware Aria Operations. the template executable, `adapter.py` has four methods:
```

Comment:
[^1] doesn't render the way I assume you want it to. Regardless, I don't think we should mention that the adapter is stateless in a footnote. I think this could use some cleaning up in general - some bits are confusing or not relevant at this point. We should talk about the architecture, but not here.